frontend/src/components/History/HistoryTable.jsx

```diff
@@ -0,0 +1,335 @@
+import {
```

Comment:
Just a broad comment, but in cases where you are completely replacing the index.jsx, we should remove those files to avoid confusion (you may have just left them in for my comparison but either way). In addition we may want to consider if the `.spec.js` files are still relevant or if we should just remove them, as they were put in for frontend tests which we never finished.